### ISSUE TITLE
ImportError: No module named 'pip.req'

### ISSUE THREAD

**Issue by doom-fr** [sqreen/PyMiniRacer#50]
### Steps to reproduce
pip install py-mini-racer

### Expected behavior
install !

### Actual behavior
setup.py", line 12, in <module>
        from pip.req import parse_requirements
    ImportError: No module named 'pip.req'

### System configuration
**Python version**:
python 3.5
pip 10.0.1 

### solution
Seems to be an incompatibility with new version of pip (>=10)
https://stackoverflow.com/a/49867265/1619521

so have to change 
`from pip.req import parse_requirements`
to
```
try: # for pip >= 10
    from pip._internal.req import parse_requirements
except ImportError: # for pip <= 9.0.3
    from pip.req import parse_requirements
```

**Comment by aviat** [comment]
Thanks, this was fixed in https://github.com/sqreen/PyMiniRacer/pull/49. Will be released soon.

**Comment by bootcher007** [comment]
Hi, I have same problem will it be fixed soon? Is there any work around?

**Comment by doom-fr** [comment]
Workaround : install previous version of pip.
Think this would work.

pip install pip==9.0.3

**Comment by ningirsu** [comment]
It should be fixed now with the last version of PyMiniRacer

**Comment by mahmutaydin1** [comment]
Hello,
In my centos 8, there is  19.3.1 version of pip. An I had this pip.req problem. To solve this, I installed pip 9 version and my problem fixed.

But  there is so many version range from 9 upto 19.. Do you think that there is another way in newer version of pip to solve this pip.req problem?

**Comment by ningirsu** [comment]
Hey, this should have been fixed in this PR https://github.com/sqreen/PyMiniRacer/pull/49

Can you send us more info regarding your problem by sending us the output of the commad `pip debug`?

Thanks

**Comment by mahmutaydin1** [comment]
thank you I solved

Sélim Menouar <notifications@github.com>, 18 Eki 2019 Cum, 23:20 tarihinde
şunu yazdı:

**Comment by chicxa** [comment]
> Hey, this should have been fixed in this PR #49
> 
> Can you send us more info regarding your problem by sending us the output of the commad `pip debug`?
> 
> Thanks

I guess this topic is closed? But i am struggling with exactly above.  Only option to downgrade? I've installed py-mini-racer. Made no difference.

Using _pip 20.1.1 from /usr/lib/python3/dist-packages/pip (python 3.8)_ ... trying to install _prometheus_

Find debug below:
pip version: pip 20.1.1 from /usr/lib/python3/dist-packages/pip (python 3.8)
sys.version: 3.8.5 (default, Aug  2 2020, 15:09:07)
[GCC 10.2.0]
sys.executable: /usr/bin/python3
sys.getdefaultencoding: utf-8
sys.getfilesystemencoding: utf-8
locale.getpreferredencoding: UTF-8
sys.platform: linux
sys.implementation:
  name: cpython
'cert' config value: Not specified
REQUESTS_CA_BUNDLE: None
CURL_CA_BUNDLE: None
pip._vendor.certifi.where(): /etc/ssl/certs/ca-certificates.crt
pip._vendor.DEBUNDLED: True
pip._vendor.WHEEL_DIR: /usr/share/python-wheels
debundled wheel versions:
  CacheControl==0.12.6
  appdirs==1.4.4
  certifi==2020.4.5.1
  chardet==3.0.4
  colorama==0.4.3
  contextlib2==0.6.0
  distlib==0.3.0
  distro==1.5.0
  html5lib==1.0.1
  idna==2.9
  ipaddr==2.2.0
  lockfile==0.12.2
  msgpack==0.6.2
  packaging==20.3
  pep517==0.8.2
  pip==20.1.1
  pkg_resources==0.0.0
  progress==1.5
  pyparsing==2.4.7
  requests==2.23.0
  resolvelib==0.3.0
  retrying==1.3.3
  setuptools==44.0.0
  six==1.15.0
  toml==0.10.1
  urllib3==1.25.9
  webencodings==0.5.1
  wheel==0.34.2

Appreciate, if you can comment.
